src/components/pages/Notes/NoteGroupNav.js

```diff
@@ -11,7 +11,7 @@ function NoteGroupNav({ children, noteGroups, addGroup, deleteGroup, updateGroup
   const [editedGroupName, setEditedGroupName] = useState(null);
   const onInputKeyDown = e => {
     if (e.keyCode === 13) {
-      updateGroupName,
+      updateGroupName(editedGroupName, editedGroupId);
```

Comment:
You rather want to pass the id first and then the name. It does not change anything codewise but is more logical to me as the id is the first information you need before you can update anything

Reply:
(oui omg on en avait déjà parlé pour les todos il me semble en plus)